### ISSUE TITLE
lsp_installer_allow_federated_servers default behavior breaks jsonls, cssls, and html

### ISSUE THREAD

**Issue by dkolb** [williamboman/nvim-lsp-installer#74]
Given the following configuration, jsonls, cssls, and html are all created in `$HOME/.local/share/nvim/lsp_servers/` in three separate directories: `vscode-langservers-extracted_jsonls`, `vscode-langservers-extracted_cssls`, `vscode-langservers-extracted_html`.  However, their LSP configs all are looking for only `vscode-langservers-extracted` for their `cmd`.  Adding `vim.g.lsp_installer_allow_federated_servers = 1` before this code fixes the issue.  Setting the global to `0` does not fix the behavior as the `cmd` for those language servers continue to use only `vscode-langservers-extracted`.

```lua
local desired_servers = {
  sumneko_lua = {
    settings = {
      Lua = {
        diagnostics = {
          globals = { 'vim' }
        },
        telemetry = {
          enable = false
        }
      }
    }
  },
  jsonls = {},
  cssls = {},
  html = {},
  bashls = {},
  yamlls = {},
  pyright = {},
  terraformls = {}
}

local lsp_installer = require'nvim-lsp-installer'

for server_name, _ in pairs(desired_servers) do
  local ok, server = lsp_installer.get_server(server_name)
  if ok then
    if not server:is_installed() then
      server:install()
    end
  end
end

lsp_installer.on_server_ready(function(server)
  local opts = desired_servers[server.name]

  server:setup(opts)
  vim.cmd [[ do User LspAttachbuffers ]]
end)
```

**Comment by williamboman** [comment]
Thanks for reporting!

> Adding `vim.g.lsp_installer_allow_federated_servers = 1` before this code fixes the issue. Setting the global to `0`

Have you tried using boolean values instead? This global only really supports boolean values, primarily due to how lua does boolean logic with numeric numbers (`:lua print(0 and "what?")`)

**Comment by williamboman** [comment]
Ah so I found an error which I've fixed in #75. Also fixed support for numeric values for the `g: lsp_installer_allow_federated_servers` option

**Comment by dkolb** [comment]
Ah, sorry I'm late.  Boolean probably would've worked. Still had my brain in vim-scripty-land.  Thanks for the quick turnaround!
